source/includes/_rars_top_referrers.md

```diff
@@ -18,14 +18,12 @@ When using the GET method, the results can be filtered using these parameters:
 |-----------|-----|---|
 | start_date | Yes | Restricts the results to those occurring on or after this date and time. |
 | end_date | Yes | Restricts the results to those occurring on or before this date and time. |
-| gmaid | Yes | Restrict results to the specified advertiser |
-|interval_size|No|Return subtotals for the given intervals within the start-end time provided. Accepts calendar_month, calendar_week, or day (Default is day)|
 
 ### Response Data Details&nbsp;
 
 
 ```shell
-curl --location --request GET 'https://api-stage.gcion.com/apgb2b-reporting/client_reports/top_referrer/TEST_1?start_date=2022-11-10&end_date=2022-11-15&interval_size=calendar_week' \
+curl --location --request GET 'https://api-stage.gcion.com/apgb2b-reporting/client_reports/top_referrer/TEST_1?start_date=2022-11-10&end_date=2022-11-15' \
```

Comment:
Avoid using the FAM route here.  This document is included in the public documentation [here](https://doc.api.localiq.com/#reporting-metric-apis-traffic-metric-apis-top-referrers) and the usage in the examples should be as an external caller (using OAuth).